src/App.css

```diff
@@ -36,3 +36,7 @@
     transform: rotate(360deg);
   }
 }
+
+body {
```

Comment:
You might want to look at this: [here](https://styled-components.com/docs/advanced)

you can wrap your app in a theme and then use generic styles throughout so you don't have to repeat `background-color: 'blue'` or `font-size: 16px`